src/services/authService.js

```diff
@@ -9,6 +9,7 @@ const signOut = () => {
  * Signs the user into the firebase application
  * @param {*} email The email of the user as a string
  * @param {*} password The password of the user as a string
+ * @returns true if the user was signed in sucessfully, false otherwise
```

Comment:
Sorry to be picky, but thought I would leave a note here :P. I've usually done JSDocs as `@returns {type} description` So here it would probably be: @returns {boolean} signed in successful or not. [Documentation](https://jsdoc.app/tags-returns.html)

Reply:
okie, good to know!